pyramid_oereb/contrib/print_proxy/mapfish_print/mapfish_print.py

```diff
@@ -73,17 +73,20 @@ def __call__(self, value, system):
         extract_as_dict = self._render(extract_record, value[1])
         feature_geometry = mapping(extract_record.real_estate.limit)
 
+        print_config = Config.get('print', {})
```

Comment:
If the variable 'print_config' is defined, it should thoroughly be used in subsequent parts of the code.